### ISSUE TITLE
dataset problems

### ISSUE THREAD

**Issue by wry980105** [ennauata/housegan#4]
Excuse me，I have applied for the LIFULL HOME datasets,but I don't know how to divide it in the training set and the test set.In addition, the dataset URL you gave was failed,I can't open it.Can you tell me how to handle it?

**Comment by ennauata** [comment]
Hello, I checked all the URL in this repository and could not find any broken one, could you tell which URL you are referring to? As mentioned in the paper we did k-fold cross validation, where each fold corresponds to graphs of certain sizes. For example, fold 1 graphs of size 1 to 3, fold 2 graphs of size 4 to 6, ...

**Comment by Ha0Tang** [comment]
@wry980105 can you share the dataset with me since I am just a student? my email is bjdxtanghao@gmail.com. Thanks.
